src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/SchemaVersion.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.overseasentitiesapi.model;
+
+/**
+* Starting at version 3.1 a new top-level 'schema_version' field with an initial value is to be included
+*/
+public enum SchemaVersion {
+    VERSION_3_1
```

Comment:
we'd want the value "3.1" in the mongo record rather than the name of the enum, so this will need a value field as well

Reply:
Yes, relates to my open comment above.

Reply:
Add string not enum to mongo.